<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>522</width>
    <height>674</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>522</width>
    <height>674</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>522</width>
    <height>674</height>
   </size>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Isaac`s Cheats</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>imgonline-com-ua-Resize-FJDPqgFmfYQzOIw.jpg</normaloff>imgonline-com-ua-Resize-FJDPqgFmfYQzOIw.jpg</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QWidget{
background-color: rgb(45, 45, 45);
}</string>
   </property>
   <widget class="QLabel" name="Int_Numbers_Only">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>261</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	color: rgb(255, 255, 255);
	font: 75 11pt &quot;MS Shell Dlg 2&quot;;
	background-color: rgb(53, 53, 53);
	border-radius: 10px;
}</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Only integer numbers!&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="margin">
     <number>10</number>
    </property>
   </widget>
   <widget class="QLabel" name="Coins">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>111</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	color: rgb(255, 255, 255);
	font: 75 11pt &quot;MS Shell Dlg 2&quot;;
	background-color: rgb(53, 53, 53);
	border-radius: 10px;
}</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Coins&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="margin">
     <number>10</number>
    </property>
   </widget>
   <widget class="QLabel" name="Bombs">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>125</y>
      <width>111</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	color: rgb(255, 255, 255);
	font: 75 11pt &quot;MS Shell Dlg 2&quot;;
	background-color: rgb(53, 53, 53);
	border-radius: 10px;
}</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Bombs&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="margin">
     <number>10</number>
    </property>
   </widget>
   <widget class="QLabel" name="Keys">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>111</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	color: rgb(255, 255, 255);
	font: 75 11pt &quot;MS Shell Dlg 2&quot;;
	background-color: rgb(53, 53, 53);
	border-radius: 10px;
}</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Keys&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="margin">
     <number>10</number>
    </property>
   </widget>
   <widget class="QLabel" name="Active_items">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>235</y>
      <width>111</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	color: rgb(255, 255, 255);
	font: 75 11pt &quot;MS Shell Dlg 2&quot;;
	background-color: rgb(53, 53, 53);
	border-radius: 10px;
}</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Active item ID&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="margin">
     <number>10</number>
    </property>
   </widget>
   <widget class="QPushButton" name="Change_coins">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>70</y>
      <width>120</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	color: rgb(255, 255, 255);
	font: 10pt &quot;MS Shell Dlg 2&quot;;
    border-radius: 6px;
	background-color: rgb(69, 69, 69);
    min-width: 80px;
}
QPushButton:hover{
	border: 2px solid rgb(50, 50, 50);
}
QPushButton:pressed {
	background-color: rgb(64, 64, 64);
}

QPushButton:flat {
    border: none; 
}

QPushButton:default {
    border-color: navy; 
}</string>
    </property>
    <property name="text">
     <string>Change</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Input_coins">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>70</y>
      <width>141</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
	font: 10pt &quot;MS Shell Dlg 2&quot;;
	border: 2px solid rgb(37, 39, 48);
	border-radius: 10px;
	color: #FFF;
	padding-left: 20 px;
	padding-right: 20 px;
	background-color: rgb(97, 97, 97);
}
QLineEdit:hover{
	border: 2px solid rgb(104, 104, 104);
}
QLineEdit:focus{
	border: 2px solid rgb(255, 255, 255);
	
	background-color: rgb(141, 141, 141);
}</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Input_bombs">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>125</y>
      <width>141</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
	font: 10pt &quot;MS Shell Dlg 2&quot;;
	border: 2px solid rgb(37, 39, 48);
	border-radius: 10px;
	color: #FFF;
	padding-left: 20 px;
	padding-right: 20 px;
	background-color: rgb(97, 97, 97);
}
QLineEdit:hover{
	border: 2px solid rgb(104, 104, 104);
}
QLineEdit:focus{
	border: 2px solid rgb(255, 255, 255);
	
	background-color: rgb(141, 141, 141);
}</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Input_keys">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>180</y>
      <width>141</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
	font: 10pt &quot;MS Shell Dlg 2&quot;;
	border: 2px solid rgb(37, 39, 48);
	border-radius: 10px;
	color: #FFF;
	padding-left: 20 px;
	padding-right: 20 px;
	background-color: rgb(97, 97, 97);
}
QLineEdit:hover{
	border: 2px solid rgb(104, 104, 104);
}
QLineEdit:focus{
	border: 2px solid rgb(255, 255, 255);
	
	background-color: rgb(141, 141, 141);
}</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Input_items">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>235</y>
      <width>141</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
	font: 10pt &quot;MS Shell Dlg 2&quot;;
	border: 2px solid rgb(37, 39, 48);
	border-radius: 10px;
	color: #FFF;
	padding-left: 20 px;
	padding-right: 20 px;
	background-color: rgb(97, 97, 97);
}
QLineEdit:hover{
	border: 2px solid rgb(104, 104, 104);
}
QLineEdit:focus{
	border: 2px solid rgb(255, 255, 255);
	
	background-color: rgb(141, 141, 141);
}</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Change_bombs">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>125</y>
      <width>120</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	color: rgb(255, 255, 255);
	font: 10pt &quot;MS Shell Dlg 2&quot;;
    border-radius: 6px;
	background-color: rgb(69, 69, 69);
    min-width: 80px;
}
QPushButton:hover{
	border: 2px solid rgb(50, 50, 50);
}
QPushButton:pressed {
	background-color: rgb(64, 64, 64);
}

QPushButton:flat {
    border: none; 
}

QPushButton:default {
    border-color: navy; 
}</string>
    </property>
    <property name="text">
     <string>Change</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Change_keys">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>180</y>
      <width>120</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	color: rgb(255, 255, 255);
	font: 10pt &quot;MS Shell Dlg 2&quot;;
    border-radius: 6px;
	background-color: rgb(69, 69, 69);
    min-width: 80px;
}
QPushButton:hover{
	border: 2px solid rgb(50, 50, 50);
}
QPushButton:pressed {
	background-color: rgb(64, 64, 64);
}

QPushButton:flat {
    border: none; 
}

QPushButton:default {
    border-color: navy; 
}</string>
    </property>
    <property name="text">
     <string>Change</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Change_items">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>235</y>
      <width>120</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	color: rgb(255, 255, 255);
	font: 10pt &quot;MS Shell Dlg 2&quot;;
    border-radius: 6px;
	background-color: rgb(69, 69, 69);
    min-width: 80px;
}
QPushButton:hover{
	border: 2px solid rgb(50, 50, 50);
}
QPushButton:pressed {
	background-color: rgb(64, 64, 64);
}

QPushButton:flat {
    border: none; 
}

QPushButton:default {
    border-color: navy; 
}</string>
    </property>
    <property name="text">
     <string>Change</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Infinity_coins">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>70</y>
      <width>84</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	color: rgb(255, 255, 255);
	font: 10pt &quot;MS Shell Dlg 2&quot;;
    border-radius: 6px;
	background-color: rgb(69, 69, 69);
    min-width: 80px;
}
QPushButton:hover{
	border: 2px solid rgb(50, 50, 50);
}
QPushButton:pressed {
	background-color: rgb(64, 64, 64);
}

QPushButton:flat {
    border: none; 
}

QPushButton:default {
    border-color: navy; 
}</string>
    </property>
    <property name="text">
     <string>∞ 99</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Infinity_bombs">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>125</y>
      <width>84</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	color: rgb(255, 255, 255);
	font: 10pt &quot;MS Shell Dlg 2&quot;;
    border-radius: 6px;
	background-color: rgb(69, 69, 69);
    min-width: 80px;
}
QPushButton:hover{
	border: 2px solid rgb(50, 50, 50);
}
QPushButton:pressed {
	background-color: rgb(64, 64, 64);
}

QPushButton:flat {
    border: none; 
}

QPushButton:default {
    border-color: navy; 
}</string>
    </property>
    <property name="text">
     <string>∞ 99</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Infinity_keys">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>180</y>
      <width>84</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	color: rgb(255, 255, 255);
	font: 10pt &quot;MS Shell Dlg 2&quot;;
    border-radius: 6px;
	background-color: rgb(69, 69, 69);
    min-width: 80px;
}
QPushButton:hover{
	border: 2px solid rgb(50, 50, 50);
}
QPushButton:pressed {
	background-color: rgb(64, 64, 64);
}

QPushButton:flat {
    border: none; 
}

QPushButton:default {
    border-color: navy; 
}</string>
    </property>
    <property name="text">
     <string>∞ 99</string>
    </property>
   </widget>
   <widget class="QPushButton" name="D6_btn">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>235</y>
      <width>84</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	color: rgb(255, 255, 255);
	font: 10pt &quot;MS Shell Dlg 2&quot;;
    border-radius: 6px;
	background-color: rgb(69, 69, 69);
    min-width: 80px;
}
QPushButton:hover{
	border: 2px solid rgb(50, 50, 50);
}
QPushButton:pressed {
	background-color: rgb(64, 64, 64);
}

QPushButton:flat {
    border: none; 
}

QPushButton:default {
    border-color: navy; 
}</string>
    </property>
    <property name="text">
     <string>D6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Inf_energy">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>480</y>
      <width>241</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	color: rgb(255, 255, 255);
	font: 10pt &quot;MS Shell Dlg 2&quot;;
    border-radius: 6px;
	background-color: rgb(69, 69, 69);
    min-width: 80px;
}
QPushButton:hover{
	border: 2px solid rgb(50, 50, 50);
}
QPushButton:pressed {
	background-color: rgb(64, 64, 64);
}

QPushButton:flat {
    border: none; 
}

QPushButton:default {
    border-color: navy; 
}</string>
    </property>
    <property name="text">
     <string>Infinity active item charge</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Dmg_btn">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>540</y>
      <width>241</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	color: rgb(255, 255, 255);
	font: 10pt &quot;MS Shell Dlg 2&quot;;
    border-radius: 6px;
	background-color: rgb(69, 69, 69);
    min-width: 80px;
}
QPushButton:hover{
	border: 2px solid rgb(50, 50, 50);
}
QPushButton:pressed {
	background-color: rgb(64, 64, 64);
}

QPushButton:flat {
    border: none; 
}

QPushButton:default {
    border-color: navy; 
}</string>
    </property>
    <property name="text">
     <string>100 Damage</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Inf_hearts_coins">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>420</y>
      <width>241</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	color: rgb(255, 255, 255);
	font: 10pt &quot;MS Shell Dlg 2&quot;;
    border-radius: 6px;
	background-color: rgb(69, 69, 69);
    min-width: 80px;
}
QPushButton:hover{
	border: 2px solid rgb(50, 50, 50);
}
QPushButton:pressed {
	background-color: rgb(64, 64, 64);
}

QPushButton:flat {
    border: none; 
}

QPushButton:default {
    border-color: navy; 
}</string>
    </property>
    <property name="text">
     <string>Infinity coins hearts</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Inf_red_hearts">
    <property name="geometry">
     <rect>
      <x>261</x>
      <y>300</y>
      <width>241</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	color: rgb(255, 255, 255);
	font: 10pt &quot;MS Shell Dlg 2&quot;;
    border-radius: 6px;
	background-color: rgb(69, 69, 69);
    min-width: 80px;
}
QPushButton:hover{
	border: 2px solid rgb(50, 50, 50);
}
QPushButton:pressed {
	background-color: rgb(64, 64, 64);
}

QPushButton:flat {
    border: none; 
}

QPushButton:default {
    border-color: navy; 
}</string>
    </property>
    <property name="text">
     <string>Infinity red hearts</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Inf_blue_hearts">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>360</y>
      <width>241</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	color: rgb(255, 255, 255);
	font: 10pt &quot;MS Shell Dlg 2&quot;;
    border-radius: 6px;
	background-color: rgb(69, 69, 69);
    min-width: 80px;
}
QPushButton:hover{
	border: 2px solid rgb(50, 50, 50);
}
QPushButton:pressed {
	background-color: rgb(64, 64, 64);
}

QPushButton:flat {
    border: none; 
}

QPushButton:default {
    border-color: navy; 
}</string>
    </property>
    <property name="text">
     <string>Infinity blue hearts</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Exit_btn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>610</y>
      <width>120</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	color: rgb(255, 255, 255);
	font: 10pt &quot;MS Shell Dlg 2&quot;;
    border-radius: 6px;
	background-color: rgb(69, 69, 69);
    min-width: 80px;
}
QPushButton:hover{
	border: 2px solid rgb(50, 50, 50);
}
QPushButton:pressed {
	background-color: rgb(64, 64, 64);
}

QPushButton:flat {
    border: none; 
}

QPushButton:default {
    border-color: navy; 
}</string>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="Inform_board">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>310</y>
      <width>211</width>
      <height>271</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTextBrowser{
	color: rgb(255, 255, 255);
	border-radius: 20px;
	border: 2px solid rgb(255, 255, 255);
	background-color: rgb(45, 45, 45);
}
</string>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt; font-weight:600;&quot;&gt; Left Ctrl + 1&lt;/span&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt; - Hook&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt; font-weight:600;&quot;&gt; Left Ctrl + 2&lt;/span&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt; - Infinity red hearts&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt; font-weight:600;&quot;&gt; Left Ctrl + 3&lt;/span&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt; - Infinity blue hearts&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt; font-weight:600;&quot;&gt; Left Ctrl + 4&lt;/span&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt; - Infinity coins hearts&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt; font-weight:600;&quot;&gt; Left Ctrl + 5&lt;/span&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt; - Infinity coins&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt; font-weight:600;&quot;&gt; Left Ctrl + 6&lt;/span&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt; - Infinity bombs&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt; font-weight:600;&quot;&gt; Left Ctrl + 7&lt;/span&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt; - Infinity keys&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt; font-weight:600;&quot;&gt; Left Ctrl + 8&lt;/span&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt; - Infinity actibe item charge&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt; font-weight:600;&quot;&gt; Left Ctrl + 9&lt;/span&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt; - 100 DMG&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Language_btn">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>10</y>
      <width>84</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	color: rgb(255, 255, 255);
	font: 10pt &quot;MS Shell Dlg 2&quot;;
    border-radius: 6px;
	background-color: rgb(69, 69, 69);
    min-width: 80px;
}
QPushButton:hover{
	border: 2px solid rgb(50, 50, 50);
}
QPushButton:pressed {
	background-color: rgb(64, 64, 64);
}

QPushButton:flat {
    border: none; 
}

QPushButton:default {
    border-color: navy; 
}</string>
    </property>
    <property name="text">
     <string>RU/EN</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Hook_btn">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>10</y>
      <width>120</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	color: rgb(255, 255, 255);
	font: 10pt &quot;MS Shell Dlg 2&quot;;
    border-radius: 6px;
	background-color: rgb(69, 69, 69);
    min-width: 80px;
}
QPushButton:hover{
	border: 2px solid rgb(50, 50, 50);
}
QPushButton:pressed {
	background-color: rgb(64, 64, 64);
}

QPushButton:flat {
    border: none; 
}

QPushButton:default {
    border-color: navy; 
}</string>
    </property>
    <property name="text">
     <string>Hook</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="Author_btn">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>630</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTextBrowser{
	color: rgb(0, 0, 0);
	border-radius: 10px;
	border: 2px solid;
	background-color: qconicalgradient(cx:0.5, cy:0.5, angle:0, stop:0 rgba(136, 106, 22, 255), stop:0.119403 rgba(166, 140, 41, 255), stop:0.233831 rgba(204, 181, 74, 255), stop:0.293532 rgba(235, 219, 102, 255), stop:0.383085 rgba(245, 236, 112, 255), stop:0.502488 rgba(209, 190, 76, 255), stop:0.57 rgba(187, 156, 51, 255), stop:0.61194 rgba(168, 142, 42, 255), stop:0.695 rgba(202, 174, 68, 255), stop:0.75 rgba(218, 202, 86, 255), stop:0.80597 rgba(208, 187, 73, 255), stop:0.835821 rgba(187, 156, 51, 255), stop:0.890547 rgba(137, 108, 26, 255));
}</string>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:7.8pt;&quot;&gt;   Made by GAVKOSHMIG Inc.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>620</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>imgonline-com-ua-Resize-FJDPqgFmfYQzOIw.jpg</pixmap>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
